apps/api/src/app/change/usecases/create-change.command.ts

```diff
@@ -0,0 +1,17 @@
+import { IsDefined, IsString } from 'class-validator';
+import { Document } from 'mongoose';
+import { CommandHelper } from '../../shared/commands/command.helper';
+import { EnvironmentWithUserCommand } from '../../shared/commands/project.command';
+
+export class CreateChangeCommand extends EnvironmentWithUserCommand {
+  static create(data: CreateChangeCommand) {
+    return CommandHelper.create(CreateChangeCommand, data);
+  }
+
+  @IsDefined()
+  item: Partial<Document>;
+
+  @IsDefined()
+  @IsString()
+  type: string;
```

Comment:
How about we create an enum here? `ChangeTypeEnum` so we can have all types tidy and structured to avoid some misspellings here?